### ISSUE TITLE
sound150@claudiux initializes with wrong panel icon

### ISSUE THREAD

**Issue by ChooChooAl** [linuxmint/cinnamon-spices-applets#5616]
### Applet version/Build date

Version 6.5.3 (2024-03-21 06:28:37)

### Cinnamon version

Cinnamon 6.0.4

### Distribution

Mint 21.3

### Graphics hardware *and* driver used

Intel UHD Graphics 620 

### Applet name and maintainer

sound150@claudiux @claudiux

### What happened?

New version was installed today by mintupdate
The initial icon has changed from the sound/speaker icon to a musical note.
![sound150@claudiux-1](https://github.com/linuxmint/cinnamon-spices-applets/assets/26556059/f83a7c42-a19b-4763-955b-4459051e5776)

Should be:
![sound150@claudiux-2](https://github.com/linuxmint/cinnamon-spices-applets/assets/26556059/70bb0ccf-147a-4d92-978a-b2a257d75c8d)

### Other information

When a restart of the desktop, either by login, system restart or cinnamon restart, the initial icon does show as sound icon.  Only after a few seconds it changes to the musical note.

**Comment by claudiux** [comment]
Thank you for reporting this issue. I know why it happens.

Open a terminal and run the following command `killall playerctld`.

Can you confirm that this solves the problem? If so, I'll make a new PR soon.

**Comment by ChooChooAl** [comment]
It fixes it for that session but as soon as I restart cinnamon it's back to the ole musical note icon.

**Comment by ChooChooAl** [comment]
Thanks.  It didn't initially but I did the 'killall playerctld' and then restarted cinnamon and it stays a speaker icon.
